query.go

```diff
@@ -32,6 +32,11 @@ var (
 	// to a peer that previously failed because of a timeout.
 	QueryPeerCooldown = time.Second * 5
 
+	// QueryRejectTimeout is the time we'll wait after sending a response to
+	// an INV query for a potential reject answer. If we don't get a reject
+	// before this delay, we assume the TX was accepted.
+	QueryRejectTimeout = time.Second
```

Comment:
We _may_ need to tune this later to increase the value (thinking about high latency links, mobile network switching, international connections, etc)

Reply:
Good point. I added this to the query options, making it configurable.